src/containers/AccountModal/AccountModal.tsx

```diff
@@ -60,6 +61,22 @@ const AccountModal = () => {
     }
   }, [viewParam, loading, isPublicView, user, toLogin]);
 
+  // handle websocket notifications
+  useEffect(() => {
```

Comment:
I think this whole thing with NotificationsController could be refactored into a hook that could have hte useEffect and useNavigate built into it.

Reply:
Yeah, that crossed my mind, but then when using it, we will need just to call it like useNotifications in the AppRoutes or somewhere, and not really use something from it, which is I guess not standard for hooks. wdyt?

Reply:
idk, I don't think it's that weird, but I would add a comment to the line where it's used.

Reply:
@dbudzins, perhaps we can change this in the services/inverify refactor (or a different cleanup PR 😄). It would be great to remove all integration-specific code from the UI and work only with generic Account, Checkout, Profile, and Subscription controllers.

For example, if we allow the account service to dispatch generic events, we can align the Cleeng and InPlayer integrations at service level. The Adobe SDK, for example, also works with global (window) callbacks that can be aligned via this method.